lib/core/network/api_service.dart

```diff
@@ -969,7 +969,7 @@ class APIService {
       );
     }
     final decodedBody = jsonDecode(response.body);
-    final itemMap = decodedBody['items']! as List<dynamic>;
+    final itemMap = decodedBody['item']! as Map<String, dynamic>;
```

Comment:
It's not really needed to put it in a variable (itemMap) first. return can also be on this line